test/e2e/specs/media/media__edit.ts

```diff
@@ -24,9 +25,10 @@ describe( DataHelper.createSuiteTitle( 'Media: Edit Media' ), function () {
 	} );
 
 	describe.each`
-		siteType      | accountName
-		${ 'Simple' } | ${ 'simpleSitePersonalPlanUser' }
-		${ 'Atomic' } | ${ 'eCommerceUser' }
+		siteType       | accountName
```

Comment:
I think this is fine for now, and is simple to scan and understand. However, in the long run, I think we should port those `describe.each` iterations to use `getTestAccountByFeature`, which would allow us to more easily support other combinations that might help catch more bugs ahead of the time (i.e it might even make sense to eventually run this as part of the GB edge tests as GB might indirectly affect this feature, too).

By the way, what kind of site does the `jetpackUser` hold? Is it a simple or AT site?

Reply:
@fullofcaffeine 

> I'd say we should get rid of the describe.each iterations and use getTestAccountByFeature instead

Yes, I do agree with this. If going down this path, we should add `@gutenberg` group to this spec file at the same time as the switch, which is what you also suggest later in the paragraph if I understand correctly.

> what kind of site does the jetpackUser hold?

I am confused by Jetpack things myself - I am not too familiar with Jetpack and the way it interacts with WordPress.com sites is really confusing. I wish there was documentation that exists (there may be that I am not aware of) which explains these things.

I checked Store Admin and didn't find the test site listed under anything that has an upgrade, so I would surmise it's a normal Simple site.

Reply:
> Yes, I do agree with this. If going down this path, we should add @gutenberg group to this spec file at the same time as the switch, which is what you also suggest later in the paragraph if I understand correctly.

Yes, I think in the long run it'd be nice to make a habit of using `getTestAccountByFeature` so that we can declare the accounts in a single place. This would move the parameterization out of the `describe.each` tables into TC builds (as env vars) that would then match the relevant entries in the central criteria table. However, as we discussed [here](https://github.com/Automattic/wp-calypso/pull/62324#discussion_r839787537), I think it's totally fine to postpone this given we're not really sure how JetPack fits into the criteria yet, and the current implementation is simple and works ;)

Reply:
I can try switching over the specs that still use `describe.each` to parametrize the tests to use `getTestAccountByFeature` in a subsequent PR, if you'd like!